bsonkit/access.go

```diff
@@ -406,3 +406,31 @@ func Multiply(doc Doc, path string, multiplier interface{}) (interface{}, error)
 
 	return field, nil
 }
+
+// Push will add the value to the array at the location in the document
+// specified by path and return the new value. If the value is missing, the
+// value is added to a new array. The type of the field may be changed as
+// part of the operation.
```

Comment:
Could you remove the last sentence since it does not apply here?

Reply:
Yes. Done!